Sources/Logging.php

```diff
@@ -501,6 +501,10 @@ function logActions($logs)
 			$smcFunc['db_free_result']($request);
 		}
 
+		$always_log = array('agreement_accepted', 'policy_accepted', 'agreement_updated', 'policy_updated');
+		if ((empty($modSettings['modlog_enabled']) && !in_array($action, $always_log)) || !isset($log['log_type']))
```

Comment:
`$action` doesn't exist. Should be `$log['action']`

Reply:
Actually... incoming PR

Reply:
https://github.com/MisterVector/SMF2.1/pull/3

Reply:
Does this fix the issue @sbulen reported about the agreement and policy acceptance not being logged?